src/constants/versions.ts

```diff
@@ -0,0 +1,6 @@
+export const VERSIONS = {
+  v1: "v1",
+  v2: "v2",
+} as const
+
+export type VERSION_NUMBERS = typeof VERSIONS[keyof typeof VERSIONS]
```

Comment:
nit: enums i think should be CamelCase and their internal values can be uppercase

https://stackoverflow.com/questions/13233137/javascript-enum-pattern-naming-convention

Reply:
or can just follow TypeScript docs

```
enum Direction {
  Up = 1,
  Down,
  Left,
  Right,
}
```

Reply:
changed to SentenceCase